x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/index.tsx

```diff
@@ -362,7 +363,8 @@ export function Detail() {
                   <WithGuidedOnboardingTour
                     packageKey={pkgkey}
                     tourType={'addIntegrationButton'}
-                    isGuidedOnboardingActive={isGuidedOnboardingActive}
+                    isTourVisible={isOverviewPage && isGuidedOnboardingActive}
```

Comment:
The Add integrations tour step was displaying at all times, regardless of which tab was open or if the old "Add agent" flyout was open. I think it makes sense to only show it when the user is on the "Overview" tab. @yuliacech let me know what you think.